types/types.go

```diff
@@ -28,6 +28,9 @@ const (
 	EnvPodNamespace = "POD_NAMESPACE"
 	EnvPodIP        = "POD_IP"
 
+	SecretAccessKey = "AWS_ACCESS_KEY_ID"
```

Comment:
`AWSxxxx = xxxx`.

Btw, the naming of access key is inconsistent with the one for the engine code. And the `SecretAccessKey` is not the access key, it's the secret key.

It's easier if we just stylize the AWS environment variable name and keep them consistent.